cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -293,6 +301,44 @@ func (impl *OSBuildJobImpl) getContainerClient(destination string, targetOptions
 	return client, nil
 }
 
+func (impl *OSBuildJobImpl) getPulpClient(targetOptions *target.PulpOSTreeTargetOptions) (*pulp.Client, error) {
+	// Read server configuration and credentials from the target options and
+	// fall back to worker config if they are not set (targetOptions take precedent).
+	// Mixing sources is allowed. For example, the server address can be
+	// configured in the worker config while the targetOptions provide the
+	// credentials (or vice versa).
```

Comment:
This should ideally go before the function definition, so that it shows up as a doc text.

Reply:
Done